### DISCUSSION TITLE
Refused to send form data to 'http://localhost:8011/login' because it violates the following Content Security Policy directive: "form-action 'self'".

### DISCUSSION THREAD

**Discussion by taochangle** [firefly-iii/firefly-iii#8662] [Firefly III]
### Support guidelines

- [X] I've read the [support guidelines](https://github.com/firefly-iii/firefly-iii/blob/main/.github/support.md)

### I've found a bug and checked that ...

- [X] ... [the documentation](https://docs.firefly-iii.org/) does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem
- [X] ... it's [definitely a Firefly III issue, not me](https://github.com/firefly-iii/firefly-iii/blob/main/.github/its_you_not_me.md)

### Description

version: 6.1.10
server:docker compose
all form page ,such as ‘/login’ ，the console logs 
`Refused to send form data to 'http://localhost:8011/login' because it violates the following Content Security Policy directive: "form-action 'self'".`

### Debug information

`Refused to send form data to 'http://localhost:8011/login' because it violates the following Content Security Policy directive: "form-action 'self'".`
macos chrome 122.0.6261.112
<img width="1685" alt="image" src="https://github.com/firefly-iii/firefly-iii/assets/7435144/20a2504c-1a4c-45f6-bcf8-f86790e12425">

### Steps to reproduce

1 Open Chrome browser
2 Visit  http://xxxx/login
3 input username and password
4 open the chrome console
5 submit the form

**Comment by JC5** [comment]
This is not a bug but a configuration issue. I presume you're running a secure server (https) yourself?

**Comment by taochangle** [comment]
okay thank u




------------------&nbsp;原始邮件&nbsp;------------------
发件人: "James ***@***.***&gt;; 
发送时间: 2024年3月11日(星期一) 下午5:57
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [firefly-iii/firefly-iii] Refused to send form data to &#39;http://localhost:8011/login&#39; because it violates the following Content Security Policy directive: &quot;form-action &#39;self&#39;&quot;. (Discussion #8662)





 
This is not a bug but a configuration issue. I presume you're running a secure server (https) yourself?
